src/components/game-board/side-panel/SidePanel.tsx

```diff
@@ -3,19 +3,21 @@ import TurnManager from './turn manager/TurnManager';
 import PlayersList from './players-list/PlayersList';
 import GameStatus from './game-status/GameStatus';
 import {TimerProvider} from '@contexts/timer.context';
-import {TURN_TIME} from '../../app/consts';
+import {GAME_TIME, TURN_TIME} from '@components/app/consts';
 
 export type SidePanelProps = ChildProps & { game: Game, updatePlayerTurn: () => void };
 
 function SidePanel({className, game, updatePlayerTurn}: SidePanelProps): React.JSX.Element {
     return (
-        <TimerProvider initialTime={TURN_TIME}>
             <div className={className}>
-            <GameStatus className='flex basis-2/3 p-5 border-2' activePlayer={game.activePlayer} nextPlayer={game.nextPlayer} updatePlayerTurn={updatePlayerTurn} />
-            <TurnManager className='flex basis-2/3 p-5 border-2'/>
+                <TimerProvider initialTime={TURN_TIME}>
```

Comment:
Settings 2 TimerProvider instance might be a problem since we have a single hook which return the data.
Please improve the TimerProvider code instead.